README.md

```diff
@@ -1,39 +1,59 @@
-# spectrum-web-components
+# Who are we?
 
 Spectrum Web Components is a future-looking project to develop Adobe Spectrum design language based around web components, ES-Modules, and modern browser standards.
 
-It will not support older browsers and will only target modern ever-green browsers that fully implement the Custom Elements V1 specification, e.g. Chrome, Firefox, Safari. Polyfills will be avoided as much as possible.
+To this end, Spectrum Web Components only targets _modern_, evergreen browsers that fully implement the Custom Elements V1 specification, e.g. Chrome, Firefox, Safari. Polyfills will be avoided as much as possible but documented if necessary.
 
 # Requirements
 
--   NodeJS >= 12.18.3
+-   NodeJS >= 14.15.3
 -   Typescript
 -   Browsers with Custom Elements V1 and Shadow DOM support, e.g. Chrome, Firefox, Safari, Edge (79+)
     -   Or appropriate [polyfills](https://github.com/webcomponents/webcomponentsjs) in older browsers.
 
-# Getting Started
+# Getting started
 
 ```bash
 git clone https://github.com/adobe/spectrum-web-components.git
 cd spectrum-web-components
 yarn
 ```
 
-The call to `yarn` will subsequently trigger scripting which will ensure that your local repo is adequately prepared to develop and run the packages in this library.
+The call to `yarn` will install and setup everything you need for developing and running the packages in this library.
 
-Typical development will involve running `yarn storybook`, `yarn test`, and `yarn docs:start` if you're making documentation changes.
+Typical development will involve running `yarn storybook`, `yarn test`, and `yarn docs:start` if you're making documentation changes (see below for additional details).
+
+## Building a new component
+
+Creating a new component from the command line can be done by running the following:
+
+```bash
+$ yarn new-package
+```
+
+This will scaffold your component's required architecture by prompting you for 2 data points - the desired name for your component and the name of the Spectrum CSS asset you will be extending.
```

Comment:
"Component" is a tricky word here. The first string is more directly the "name of the package" or the "name of the directory in which it will live", but maybe that's splitting hairs?

Reply:
`extended` feels weird... "CSS asset from which you will be building" or "CSS asset you will be implementing"? Words are hard.

Reply:
```suggestion
This will scaffold your component's required architecture by prompting you for 2 data points - the desired name for your package and the name of the Spectrum CSS asset from which you will be building.
```